.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
```

Comment:
Actually we probably want `sudo: false` here, so that we can [use their container-based infrastructure](https://docs.travis-ci.com/user/workers/container-based-infrastructure/)